### DISCUSSION TITLE
osu!mania mobile

### DISCUSSION THREAD

**Discussion by DarkGradient** [ppy/osu#29700] [Ideas]
Why does osu!mania uses weird input boxes?
![Screenshot_2024-09-05-12-04-28-680_com roblox client](https://github.com/user-attachments/assets/3e7f507f-6450-446f-bf64-9793bac1b794)
Roblox mania feels more suitable than osu
![Screenshot_2024-09-05-12-06-09-722_sh ppy osulazer](https://github.com/user-attachments/assets/8710af35-43d0-4a68-befa-403e21e72de3)
Also how i see, osu doesn't seems to be registered taps correctly.
At least i wanna ask, why everything in gameplay is so small

**Comment by peppy** [comment]
You can change the size of things using the UI scale setting.

**Reply by DarkGradient** [reply]
Unfortunately this doesn't help, the osu interface still doesn't feel optimized for a mobile device.
using 1.6 max scale.

**Reply by Tenexxt** [reply]
It doesn't feel like it's optimised because it isn't.
The best example is the mod menu where half of the screen is flooded with UI that's not really useful.
As for the gameplay boxes, they are/will be getting worked on to be adjustable, and also you will be able to edit the position and rotation of the mania playfield.
I didn't hear of a date where both of these will be available, but its something i guess.
